pwndbg/arguments.py

```diff
@@ -53,15 +53,26 @@ def get_syscall_name(instruction):
         return None
 
     syscall_register = pwndbg.lib.abi.ABI.syscall().syscall_register
+    syscall_arch = pwndbg.gdblib.arch.current
 
-    # If we are on x86/x64, return no syscall name for other instructions than syscall and int 0x80
+    # On x86/x64 `syscall` and `int <value>` instructions are in CS_GRP_INT
+    # but only `syscall` and `int 0x80` actually execute syscalls on Linux.
+    # So here, we return no syscall name for other instructions and we also
+    # handle a cse when 32-bit syscalls are executed on x64
```

Comment:
Does "cse" stand for something or did you mean "case"?

Reply:
typo: should be 'case' :)